l10n_it_delivery_note_rma/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Return Merchandise Authorization Management - Link with Delivery Notes",
```

Comment:
Please adapt it to the following convention: https://www.odoo-italia.org/documentazione/14.0/sviluppo/guidelines.html#titolo-del-modulo

Reply:
@tafaRU può andare "ITA - Creazione RMA da DDT (Delivery Note)" ?

Reply:
:+1:

Reply:
Ok